src/ImageSharp/ImageSharp.csproj

```diff
@@ -25,7 +25,7 @@
     <DebugType Condition="$(codecov) == ''">portable</DebugType>
     <DebugSymbols>True</DebugSymbols>
     <Features>IOperation</Features>
-    <LangVersion>7.3</LangVersion>
+    <LangVersion>preview</LangVersion>
```

Comment:
This could be an issue.

Reply:
This works with the v2.2.202 of the dotnet SDK that just got published. And will also work when the Visual Studio 2019 images go live.

Reply:
@dlemstra noticed an issue with the "wildcard" settings. Can't remember what it was, but we decided to go explicit. Even without understanding it, it seems safer to me if we want interop across different IDE-s / SDK-s.